hxinsight-extension/src/main/java/org/alfresco/hxi_connector/hxi_extension/service/model/Agent.java

```diff
@@ -34,7 +34,7 @@
 @NoArgsConstructor
 public class Agent
 {
-    private String agentId;
+    private String id;
```

Comment:
I see quite a lot of changed fields but only agentId -> id has been changed here. I'm guessing we don't need to map any of them at the moment?

Reply:
For now we need only `id`, `name`, `description`.